landing-pages/site/content/en/ecosystem/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Ecosystem"
+linkTitle: "Ecosystem"
+menu:
+  main:
+    weight: 30
+---
+
+&nbsp;
+&nbsp;
+
+# Ecosystem
+
+Feel free to contribute any resources or tools that you find useful when learning or using Airflow.
+
+&nbsp;
+
+## Learning resources
+
+[Apache Airflow YouTube Channel](https://www.youtube.com/channel/UCSXwxpWZQ7XZ1WL3wqevChA) - Official YouTube Channel
+
+[Airflow Summit](https://airflowsummit.org/) - Online conference for Apache Airflow developers
+
+[Awesome Apache Airflow](https://github.com/jghoman/awesome-apache-airflow) - Curated list of resources about Apache Airflow
+
+[The Complete Hands-On Introduction to Apache Airflow](https://www.udemy.com/course/the-complete-hands-on-course-to-master-apache-airflow) by Marc Lamberti on Udemy
+
+[Apache Airflow: Complete Hands-On Beginner to Advanced Class](https://www.udemy.com/course/apache-airflow-course) by Alexandra Abbas on Udemy
+
+&nbsp;
+
+## Airflow-As-A-Service
+
+[Astronomer](https://www.astronomer.io/) - Managed Apache Airflow service on Astronomer Cloud
+
+[Google Cloud Composer](https://cloud.google.com/composer) - Managed Apache Airflow service on Google Cloud Platform
+
+&nbsp;
+
+### Tools integrating with Airflow
```

Comment:
I assume this should be `##` rather than `###` (to be consistent with another two sections above)?

Reply:
That's right, I'm updating that